.github/workflows/test.yml

```diff
@@ -85,3 +85,21 @@ jobs:
       - name: Run unit tests with tox (macOS)
         if: startsWith(matrix.platform, 'macos')
         run: tox -e py-unit
+
+  man:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          python -m pip install tox tox-gh>=1.2
```

Comment:
You are not using `tox-gh` in the workflow. `tox-gh` only affects bare `tox` calls without an `-e` argument.